### PR TITLE
Weapons storage tweaks

### PR DESCRIPTION
**Pull request by sumghai**
Summary of changes:
- Add wooden Weapons Cabinet
- Rebalance/increase weapons storage max mass/bulk capacities (#14)
- Fix: Hide weapons locker contents when running Combat Extended (#15)
- Various minor housekeeping tasks

### CODE REVIEW

**Comment by lilwhitemouse** [comment]
Never saw these until just now.  Thanks - will see if I can figure out how to manage these, when I have already made some of the changes...

**Comment by lilwhitemouse** [comment]
"Correct convention for building UI icons in the architect menu is to place them inside the Textures/UI folder"  - what is the architect menu?

**Comment by sumghai** [comment]
The architect menu is the menu at the bottom-left corner of the screen where players choose which buildings/furniture to build on the map.

I'm just going through the list of conflicting files just now, making sure your final changes takes precedence over mine.

**Comment by sumghai** [comment]
Okay, just resolved all the merge conflicts!

I pretty much went with your rationale for the vanilla mass capacity values and increased storage time. The only thing on my end was adding the wooden weapons cabinets, since it was strangely absent all this time.

You can go ahead and merge this PR when you're ready :)

File: _Mod/LWM.DeepStorage/Patches/Compatibility_StockpileForDisaster.xml [hunk 1]
```diff
@@ -4,38 +4,40 @@
...
+		<mods>
+			<li>[1.0]-StockpileForDisaster</li>
+		</mods>
+		<nomatch Class="LWM.DeepStorage.PatchMessage">
+			<message>WTF</message>
+		</nomatch>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="LWM_DeepStorage"]/inspectorTabs</xpath>
```
Comment (lilwhitemouse) [inline _Mod/LWM.DeepStorage/Patches/Compatibility_StockpileForDisaster.xml]: What am I writing?

Comment (sumghai) [reply _Mod/LWM.DeepStorage/Patches/Compatibility_StockpileForDisaster.xml]: I'm not familiar with StockpileForDisaster, so all I did was just cleanup your indents.

**Comment by lilwhitemouse** [comment]
If there are any weird comments here, it's probably me talking to myself as I try to use GitHub.

**Comment by sumghai** [comment]
> The vanilla UI icons all have icons named normal/path/to/icon_MenuIcon, as far as I can tell?

Interesting. Many mods I've seen use the Textures/UI menuicon file path method.

> GitHub is not easy to handle correctly (unless perhaps one is very used to it). We should probably have separated these into 7 different pull requests, and for all I know, there is a way to do that, but I can't find it.

My advice here would be to accept the pull request as-is (including some further, minor changes I made for #14 and #20)

Pull requests don't need to be split up to cater for individual commits, nor should they be held back until made perfect.

**Comment by lilwhitemouse** [comment]
Alas, I would like the keep my spacing in the XML ><

**Comment by sumghai** [comment]
Okay, what else needs to happen here?

I've already made all the requested changes, and even left your XML spacing alone.

**Comment by lilwhitemouse** [comment]
I am unwilling to move the UI files to a special directory of their own:
I have artwork from multiple people and, because they have differing licenses, want to keep them clearly separate.
So, e.g., commit a5a6e45bcd880fcbf2aec34583acbe7320f3030f (https://github.com/lilwhitemouse/RimWorld-LWM.DeepStorage/pull/18/commits/a5a6e45bcd880fcbf2aec34583acbe7320f3030f).

**Comment by sumghai** [comment]
Okay, that's fine.

**Pull request merged**